.circleci/config.yml

```diff
@@ -585,3 +634,16 @@ workflows:
             branches:
               ignore:
                 - /.*/
+      - publish-prerelease:
+          context: org-global
+          requires:
+            - pr-tools/pr-title-check
+            - test-lint
+            - test-unit
+            - test-coverage
+            - vulnerability-check
+            - audit-licenses
+          filters:
+            branches:
+              only:
+                - /(major|minor|patch)/.*/
```

Comment:
`publish-prerelease` added above is an implementation for the [pre-release branches](https://docs.mojaloop.io/community/standards/creating-new-features.html#prerelease-branches) which I proposed recently.

This job will be useful to have in many other, so I am opening this issue https://github.com/mojaloop/ci-config/issues/38 to move the code to a central place.

Reply:
Thanks @kalinkrustev , really appreciate your contribution here. 

There was no opposition on the DA and I had a discussion with James on this and we're ok to move forward with this. Thanks again!